pkg/ir/unspecified.go

```diff
@@ -42,13 +42,18 @@ func (s *Spec) ComputeBlockedSubnets() []ID {
 		// cidr segments which include the subnet
 		cidrSegments := []ID{}
 		for segmentName, cidrSegmentDetails := range s.Defs.CidrSegments {
+			subnetInSegment := false
```

Comment:
Consider using [a labeled break statement](https://stackoverflow.com/questions/51996175/how-to-break-out-of-nested-loops-in-go) instead.
Another option is a labeled `continue` statement.